cloud/pkg/controller/controller/downstream.go

```diff
@@ -456,13 +456,14 @@ func (dc *DownstreamController) Start() error {
 
 // Stop DownstreamController
 func (dc *DownstreamController) Stop() error {
-	log.LOGGER.Infof("stop downstream controller")
+	log.LOGGER.Warnf("Prepare to stop downstream controller...")
```

Comment:
Info is enough here, stop controller is normal operation

Reply:
@fisherxu but we do not want controller to stop.   if controller stoped, there must be something wrong.

Reply:
> if controller stoped, there must be something wrong

no, If user let it stoped, it's nothing wrong. And If controller stop because something wrong, should have other error log. Ref the k8s controller: #[deployment controller](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/deployment/deployment_controller.go#L152)

Reply:
@fisherxu ok, thank you